### PR TITLE
Controller

### PR DESCRIPTION
**Pull request by Ursaaaaaa**
- Use controllers to handle requests and render empty views.
- Understand Rails naming conventions.
- Use params from browser requests in a safe way.

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ramrod433, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitlow was correctly used


# Required Changes ♻️

- [ ] @ramrod433 I didn't manage to run your project, I encountered an issue with your migration files, below is a screenshot of the error, kindly take a look on it:
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Ursaaaaaa** [comment]
@danmainah I have no problems on my end. Is it something from your end?

**Comment by danmainah** [comment]
> @danmainah I have no problems on my end. Is it something from your end?

@ramrod433 let me recheck it again.

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
+  get 'users/index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
```
Comment (nicupop729) [inline config/routes.rb]: It is not advisable to have the routes set twice. Also, these routes are not correctly set.

- [x] Please consider removing these routes as you only need those below, which are correctly set.

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hello @ramrod433  👋,

_This is an additional review as a result of the second opinion request._

I am happy to let you know that I could run and test your project! :+1: 

However, the instructions in the README file are not completed. So please, I emphasize that you need to edit the instructions on how a user should clone your repo, otherwise, other reviewers may not run your app.

Please also check the inline comment I left in your project.

When ready, please ask for another review.

Keep rocking 🚀!

File: config/routes.rb (hunk 1 above)
Comment (Ursaaaaaa) [reply config/routes.rb]: Noted. Thank you

**Review by alexander16108 — APPROVED** [review]
Hi @ramrod433 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**